hw1/bashcommands.py

```diff
@@ -0,0 +1,79 @@
+from collections import deque
+
+
+def for_each_argument(f):
```

Comment:
Let's also add Typing to all functions in this PR so that you get a chance to try typing Callables.